docs/userguides/contracts.md

```diff
@@ -171,13 +171,33 @@ def set_number(num: uint256):
     self.myNumber = num
 ```
 
-You can call those functions by doing:
+Notice the contract has both an external pure method and an external method that modifies state.
+In EVM languages, methods that modify state require a transaction to execute because they cost money.
+Modifying the storage of a contract requires gas and thus requires a sender with enough funding.
+Contract calls, on the other hand, are read-operations and do no cost anything.
+Thus, calls do no require specifying a `sender=` in Ape.
+
+At the RPC level, Ethereum calls are performed using the `eth_call` RPC and transactions are performed using the `eth_sendTransaction` or `eth_sendRawTransaction` RPCs.
+
+The following examples demonstrate both making a call and invoking a transaction to a deployed version of the contract above.
+First, here is an example of invoking a function (creating a transaction):
 
 ```python
-assert contract.get_static_list() == [1, 2, 3]
+from ape import accounts, Contract
 
-# Mutable calls are transactions and require a sender
-receipt = contract.set_number(sender=dev)
+sender = accounts.load("<ALIAS>")
```

Comment:
I usually like to call this something like `account` so `sender=account` appears more demostrative